techniques/general/GXX_motionactuationfailure.html

```diff
@@ -14,58 +14,48 @@ <h2>Metadata</h2>
 		</section>
 		<section id="applicability">
 			<h2>Applicability</h2>
-			<p>Use this technique on web pages that detect device or user motion (such as shaking or tilting) and use these motions as a means of input.</p>
-			<p>This technique does not relate to movement of users through space as registered by geolocation sensors or beacons, or events observed by the device other than intentional gesturing by the user. It also does not cover indirect motion associated with operating a keyboard, pointer, or assistive technology. </p>
-      <p>This technique relates to Success Criterion 2.5.4: Motion Actuation (Failure).</p>
+			<p>Content using technologies that can support the detection of device or user motion such as shaking or tilting and use these motions as a means of input.</p>
 		</section>
 		<section id="description">
 			<h2>Description</h2>
-			<p>This describes the failure condition that results when motion actuation cannot be deactivated.  People who may accidentally activate sensors due to tremors or other motor impairments need the ability to turn off motion actuation to prevent such accidental triggering of functions.  </p>
+			<p>This describes the failure condition that results when motion actuation can not be deactivated.  People who may accidentally activate sensors due to tremors or other motor impairments need the ability to turn off motion actuation to prevent such accidental triggering of functions.  </p>

```

Comment:
wondering it the "can not be deactivated" needs some extra clarification that the deactivation/option needs to be part of the content, and that deactivating it at OS level doesn't count? (or does it?)

Reply:
Do we need to specify? Isn't there a (future) possibility that the user could disable at the OS level which the website can pick up on?

Reply:
in future perhaps, but for now it raises the usual questions of "if the content doesn't do anything, but on iOS or whatever the user can do it at OS level...is that a pass?"

Reply:
(though then of course the usual answer is "not unless ALL user agents/OS provide this functionality", most likely)

Reply:
That does depend on 'accessibility supported' and your conformance claim, but as a failure technique it just needs to say 'Check if there is a user setting which disables the motion detection'.

Reply:
so if there's an OS/UA level setting to disable motion controls, and i say i'm only conformant on that particular OS/UA, then it's a pass without me doing anything as i satisfy #3 ?

Reply:
I think that would be appropriate, yes. Of course, you'd need another mechanism to trigger the function, but that's under other SCs (2.1.1 I assume).

Reply:
it's probably something more for understanding to clarify...will chew this over when i'm not poolside on hols ;)